README.md

```diff
@@ -1,40 +1,55 @@
-# Azure Sentinel 
-In the world of technology, cyber-security is the #1 priority for any company. And with the almost infinite amount of threats any company can face it seems almost impossible to do. In addition to that they are many companies that are selling partial solutions  but it gets cumbersome to make sure which version of what works with each other. It woulds be easier if all of the solutions were in one service. Microsoft heard that call and answered that call with  Azure Sentinel.    
+# Azure Sentinel
+In the world of technology, cybersecurity is the #1 priority for any company. And with the almost infinite amount of threats any company can face, it seems almost impossible to do. In addition to that, there are many companies that are selling partial solutions but it gets cumbersome to make sure which version of what works with each other. It woulds be easier if all of the solutions were in one service. Microsoft heard that call and answered that call with Azure Sentinel.
 
-Azure Sentinel is a SIEM (Security Event Information Management) and a SOAR Security Orchestrated Response) that Microsoft has created. Azure Sentinel  delivers security and threat intelligence and proving a single solution for alert detection, threat visibility, proactive hunting and threat response
+Azure Sentinel is both a SIEM (Security Information and Event Management) and a SOAR (Security Orchestration, Automation, and Response) solution that Microsoft has created. Azure Sentinel delivers security and threat intelligence and proving a single solution for alert detection, threat visibility, proactive hunting, and threat response.
```

Comment:
You say "delivers threat intelligence", but later say that Sentinel doesn't have it baked in. Not sure if you want to leave that out or caveat it here.